packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -59,6 +59,45 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       })
     })
 
+    it('can translate, create, and publish a draft of a policy intent', async () => {
+      const draft: NoParkingIntentDraft = {
+        intent_type: 'no_parking',
+        rule_fields: {
+          geographies: [],
+          days: null,
+          start_time: null,
+          end_time: null
+        },
+        policy_fields: {
+          name: 'aname',
+          description: 'aname',
+          provider_ids: [],
```

Comment:
For the sake of testing it might be good to include a provider ID here and assert it, just in case there's a bug that causes provider_ids in the intent to not be correctly associated with the created policy.